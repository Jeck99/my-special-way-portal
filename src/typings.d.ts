/* SystemJS module definition */
declare var module: NodeModule;
interface NodeModule {
  id: string;
}

declare const MSW_HOT_CONFIG;
