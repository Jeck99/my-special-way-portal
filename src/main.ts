import { enableProdMode } from '@angular/core';
import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';

import { AppModule } from './app/app.module';
import { environment } from './environments/environment';

if (typeof MSW_HOT_CONFIG !== 'undefined' && MSW_HOT_CONFIG) {
  environment.hotConfig = MSW_HOT_CONFIG;
}
if (environment.production) {
  enableProdMode();
}
platformBrowserDynamic().bootstrapModule(AppModule)
  .catch((err) => console.log(err));
