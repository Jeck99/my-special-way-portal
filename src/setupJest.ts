import 'jest-preset-angular';
import 'jest-zone-patch';
import 'jest-localstorage-mock';
import './app/rxjs-imports';
