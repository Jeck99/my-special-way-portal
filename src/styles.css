/* You can add global styles to this file, and also import other style files */
.my-beautiful-icon {
    background: rgb(56, 155, 255) !important;
    border-radius: 50%;
}