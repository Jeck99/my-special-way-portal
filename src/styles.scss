/* You can add global styles to this file, and also import other style files */
@import 'src/app/map/map.global.scss';