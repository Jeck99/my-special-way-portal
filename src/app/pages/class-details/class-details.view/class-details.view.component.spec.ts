import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { ClassDetailsViewComponent } from './class-details.view.component';
import { NO_ERRORS_SCHEMA } from '@angular/compiler/src/core';

describe('ClassDetailsComponent', () => {
  let fixture: ComponentFixture<ClassDetailsViewComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        ClassDetailsViewComponent,
      ],
      schemas: [NO_ERRORS_SCHEMA],
    })
      .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ClassDetailsViewComponent);
    fixture.detectChanges();
  });

  it.only('should match current snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
