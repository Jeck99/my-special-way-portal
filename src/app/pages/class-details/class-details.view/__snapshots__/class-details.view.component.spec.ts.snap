// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClassDetailsComponent should match current snapshot 1`] = `
<app-class-details-view>
  <div
    class="main-content"
  >
    <div
      class="container-fluid"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12 col-md-12 col-sm-12"
        >
          <table
            class="mat-table schedule-table"
            role="grid"
          >
            <thead>
              <tr
                class="mat-header-row"
                role="row"
              >
                <th
                  class="mat-header-cell cdk-column-position mat-column-position"
                  colspan="2"
                  role="columnheader"
                >
                  שעה
                </th>
                
              </tr>
            </thead>
            <tbody>
              
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</app-class-details-view>
`;
